client/src/locales/en.json

```
@@ -856,6 +879,14 @@
 		"pending": "Pending...",
 		"up": "Your site is up."
 	},
+	"statusPage": {
```

Biome (2.1.2)

[error] The key statusPage was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)